torch_tvm/compiler.cpp

```diff
@@ -109,11 +108,10 @@ tvm::relay::Expr TVMCompiler::convertToRelay(
   if (val.isIntList()) {
     tvm::Array<tvm::relay::Expr> tuple_elems;
     for (const auto& elem : val.toIntList()) {
-      auto x = tvm::runtime::NDArray::Empty(
-          {}, tvm::runtime::String2TVMType("int32"), ctx);
-      TORCH_CHECK(elem <= std::numeric_limits<int32_t>::max());
-      TORCH_CHECK(elem >= std::numeric_limits<int32_t>::lowest());
-      reinterpret_cast<int32_t*>(x->data)[0] = elem;
+      auto x = tvm::runtime::NDArray::Empty({}, tvm::Int(64), ctx);
+      TORCH_CHECK(elem <= std::numeric_limits<int64_t>::max());
```

Comment:
this shouldn't be necessary (elem is a signed 64bit int)

Reply:
Aah. makes sense. Too many things in the same day.